.github/workflows/cd-firebase-testnet.yml

```
@@ -18,9 +18,9 @@ jobs:
 
     name: Node ${{ matrix.node }} CD Firebase testnet
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@df4cb1c069e1874edd31b4311f1884172cec0e10 # v6.0.3
       - name: Use Node.js ${{ matrix.node-version }}
```

actionlint (1.7.12)

[error] property "node-version" is not defined in object type {node: string}

(expression)